docs/book/src/reference/metrics.md

```diff
@@ -3,6 +3,53 @@
 By default, controller-runtime builds a global prometheus registry and
 publishes a collection of performance metrics for each controller.
 
+## Exporting Metrics for Prometheus
+
+The project can be enabled to be configured by using `MonitorService` resources from
+the Prometheus Operator. Following the steps to do this setup.
+
+1. Install Prometheus Operator. You can follow up the steps in [Kube Prometheus](https://github.com/coreos/kube-prometheus) project.
+2. Then, enable the project to export the metrics by uncommenting the line `- ../prometheus` in the `config/default/kustomization.yaml` to allow 
+it creates the `ServiceMonitor` resource which is used to do this export. Following an example.
+
+```yaml
+# [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
+- ../prometheus
+```
+
+Note, that now when you install your project in the cluster by the following steps it will create the
+`Service` and `Monitor Service` to export the metrics. 
+
+```yaml
+$ make install
+$ kubectl apply -f config/samples/
+$ make docker-build docker-push IMG=<user>/<project:tag> # use your registry
+$ make deploy IMG=cmacedo/monitor:0.0.1
+```
+
+Then, see that you will be able to check the `service/monitor-controller-manager-metrics-service` deployed as the following example.
+
+```yaml
+$ kubectl get  all -n <project>-system
```

Comment:
I see that you changed namespace prefix to "project-v2" but here only project" is being used. More importantly, do you think "get all" is required here? If we want to just show the service, can we just do "get service"? In addition, it may be helpful to show "ServiceMonitor" resource here.

Reply:
Hi @draghuram, 

Thank you for your review. 

> I see that you changed namespace prefix to "project-v2" but here only project" is being used. 

Shows that the doc is outdated. I'd like to have a confirmation from the maintainers over that if you do not mind.

>  More importantly, do you think "get all" is required here? If we want to just show the service, can we just do "get service"? In addition, it may be helpful to show "ServiceMonitor" resource here.

👍